build/docs.js

```diff
@@ -1,31 +1,41 @@
 console.log('Building Leaflet documentation with Leafdoc ...');
 
-var LeafDoc = require('leafdoc');
-var doc = new LeafDoc({
-	templateDir: 'build/leafdoc-templates',
-	showInheritancesWhenEmpty: true,
-	leadingCharacter: '@'
-});
+var runs = {
+	'api-docs': 'reference.html',
+	'graphviz-uml-bare': 'class-diagram-bare.dot',
+	'graphviz-uml-full': 'class-diagram-full.dot'
+};
+
+Object.entries(runs).forEach(function (run) {
+
+	var LeafDoc = require('leafdoc');
+	var doc = new LeafDoc({
+		templateDir: 'build/leafdoc-templates/' + run[0] + '/',
+		showInheritancesWhenEmpty: true,
+		leadingCharacter: '@'
+	});
 
-// Note to Vladimir: Iván's never gonna uncomment the following line. He's
-// too proud of the little leaves around the code.
-// doc.setLeadingChar('@');
+	// Note to Vladimir: Iván's never gonna uncomment the following line. He's
+	// too proud of the little leaves around the code.
+	// doc.setLeadingChar('@');
```

Comment:
Are this lines still correct? In the options `leadingCharacter: '@'` is defined.

Reply:
Well, ***I*** didn't uncomment the line, so it's *technically* correct.

But, yes, that comment is superfluous right now.